package.json

```diff
@@ -37,9 +37,11 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "phantomjs-prebuilt": "^2.1.7",
-    "react": "^15.0.0",
```

Comment:
The use case for this is developing this component with `npm link`. The problem is that in the other case you can have 2 versions of react and react doesn't like that.

To me it looks like it only complicates things for people who want to develop react-onclickoutside. Anyone who uses this, will already have react.

I will close this at the moment, because it looks like I'm the only one with the issue at the moment.